src/components/modals/ListModal.tsx

```diff
@@ -94,7 +95,8 @@ export function ListModal<T>({
         keyExtractor={(_, i) => `${i}`}
         onViewableItemsChanged={onViewableItemsChanged}
       />
-      {closeArrow && <ModalCloseArrow onPress={handleCancel} />}
+      <GradientFadeOut />
+      {!closeArrow ? null : <ModalCloseArrow onPress={handleCancel} />}
```

Comment:
Fadeout should just be put into the close arrow

Reply:
Adding Fadout to all `ModalCloseArrow` components will introduce a bug where the `GradientFadeout` covers overlay buttons in all `ButtonsModal` components, since those use `ModalCloseArrow`, in additional to one-off modals such as the `ScanModal`. This visual feature (in it's current state) is just meant for list modals.

Reply:
In that case, put the fadeout underneath the close arrow.